Fixup ifplugd for 

This may be partially from our buysbox implementation.

--- a/policy/modules/services/ifplugd.te
+++ b/policy/modules/services/ifplugd.te
@@ -70,3 +70,9 @@ sysnet_signal_dhcpc(ifplugd_t)
 optional_policy(`
 	consoletype_exec(ifplugd_t)
 ')
+
+# Busybox creates /run/ifstate.new and then renames to ifstate
+files_create_etc_runtime(ifplugd_t, "ifstate.new")
+# It tries to modpribe netdev-eth0, which we don't really need.
+kernel_dontaudit_request_load_module(ifplugd_t)
+dontaudit ifplugd_t self:capability sys_module;
